scripts/proxy/shared/models.py

```
@@ -0,0 +1,67 @@
+            "Edit": lambda: f"Edit -> {_get_path('file_path')}",
+            "Read": lambda: f"Read -> {_get_path('file_path')}",
+            "Write": lambda: f"Write -> {_get_path('file_path')}",
+            "Bash": lambda: f"Bash -> {(self.args.get('command') or '?')[:40]}",
+            "Grep": lambda: f'Grep -> "{(self.args.get("pattern") or "?")[:20]}"',
+            "Glob": lambda: f"Glob -> {(self.args.get('pattern') or '?')[:30]}",
+            "Task": lambda: f"Task -> {(self.args.get('description') or '?')[:30]}",
+            "TodoWrite": lambda: "TodoWrite -> updated todos",
+        }
+        return extractors.get(self.name, lambda: self.name)()
+
+    def matches_search(self, query: str, nested: bool = True) -> bool:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)